Description: Remove hostname and build time from version as they make the build unreproducible
Author: Jelmer Vernooij <jelmer@debian.org>
Status: not-forwarded

diff --git a/configure.ac b/configure.ac
index 5b79a9e..5ef9b51 100644
--- a/configure.ac
+++ b/configure.ac
@@ -680,7 +680,7 @@ cat > include/newversion.h.in <<EOF
 #ifndef VERSION_HIDDEN
 #define VERSION_HIDDEN
 #endif
-VERSION_HIDDEN const char *heimdal_long_version = "@([#])\$Version: $PACKAGE_STRING by @USER@ on @HOST@ ($host) @DATE@ \$";
+VERSION_HIDDEN const char *heimdal_long_version = "@([#])\$Version: $PACKAGE_STRING \$";
 VERSION_HIDDEN const char *heimdal_version = "AC_PACKAGE_STRING";
 EOF
 
